modules/templates/conf.py_t

```diff
@@ -71,7 +71,8 @@ extensions = ['sphinx.ext.githubpages',
     'sphinx.ext.autodoc',
     'sphinx.ext.napoleon',
     'javasphinx',
-    'sphinxcontrib.apiblueprint']
+    'sphinxcontrib.apiblueprint',
+    'blog']
```

Comment:
Instead of this, I think you can use `extensions.blog` as then it shouldn't require adding extensions to sys.path. Please confirm this and if true make the changes. Other than that looks good.